layers/+misc/multiple-cursors/packages.el

```diff
@@ -34,3 +34,27 @@
           (define-key state-map (car keybinding) (cdr keybinding))))
       (add-hook 'prog-mode-hook 'turn-on-evil-mc-mode)
       (add-hook 'text-mode-hook 'turn-on-evil-mc-mode))))
+
+(defun multiple-cursors/init-multiple-cursors ()
+  (use-package multiple-cursors
+    :defer t
+    :init
+    (progn
+      (spacemacs/declare-prefix "sm" "multiple-cursors")
+      (spacemacs/declare-prefix "sms" "specials")
+      (spacemacs/set-leader-keys "smm" 'mc/mark-more-like-this-extended)
```

Comment:
The key bindings can be bound under a single `(spacemacs/set-leader-keys`:

https://github.com/syl20bnr/spacemacs/blob/b4907d0540bae38588ed50fad063883c76c41867/layers/%2Bchat/erc/packages.el#L50-L54